modules/sops/templates/subs.py

```diff
@@ -0,0 +1,17 @@
+from sys import argv
+
+target = argv[1]
+subst = "@subst@"
+
+with open(target) as f:
+    content = f.read()
+
+with open(subst) as f:
+    subst_pairs = f.read().splitlines()
+
+for pair in subst_pairs:
+    placeholder, path = pair.split()
+    with open(path) as f:
+        content = content.replace(placeholder, f.read())
+
+print(content)
```

Comment:
Would be nice to wrap in this in a `def main() -> None:` function so that it's type checked by mypy.

Reply:
mhm. might be type checked anyway but I usually prefer not to introduce global variable as it might become ugly as the program grows.